workflow/worklog.md

```diff
@@ -18,3 +18,20 @@ Initial flattening of SNOMED conditions in aou synthetic:
 Need to export postgres password and service account credentials file.
 ```
 ./gradlew workflow:execute -DmainClass=bio.terra.tanagra.workflow.CopyBigQueryDatasetToPostgres -Dexec.args="--underlayYaml=api/src/main/resources/underlays/synpuf.yaml --cloudSqlInstanceName=broad-tanagra-dev:us-central1:tanagra-cloudsql-2cd088adcd745a91 --cloudSqlDatabaseName=indexes --cloudSqlUserName=tanagra --runner=dataflow --project=broad-tanagra-dev --region=us-central1 --serviceAccount=tanagra@broad-tanagra-dev.iam.gserviceaccount.com"```
+```
+
+## Generate hierarchy tables for all concept domain-based entities
+Fetch the tanagra-dev SA key file from Vault. (You may need to be on the Broad VPN to do this.)
```

Comment:
Could you leave a link to the vault setup? I never remember how to initialize this

Reply:
Added some links to setting up Vault credentials. Let me know if that's not what you meant about vault setup.